django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+from django.conf import settings
+from django.db import models
+
+
+class Comment(models.Model):
+    package = models.ForeignKey(
+        "repository.Package",
+        on_delete=models.CASCADE,
+        related_name="comments",
+    )
+    community = models.ForeignKey(
```

Comment:
Rather than including community as a direct reference, see above. If we use `PackageListing` as the main reference, the community relation is already implied